src/lib.rs

```diff
@@ -58,17 +58,18 @@ mod handshake;
 ))]
 pub mod stream;
 
-use std::io::{Read, Write};
+use std::{
+    io::{Read, Write},
+    pin::Pin,
+    task::{ready, Context, Poll},
```

Comment:
Needs bumping MSRV to 1.64, which seems fine

Reply:
I'm not sure what else needs to be changed to modify MSRV.

Reply:
I don't think anything else, thanks :)